.github/workflows/add_release_label.yml

```diff
@@ -0,0 +1,62 @@
+name: Add release label to PR and linked issues when PR gets merged
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types:
+      - closed
+
+jobs:
+  add_release_label:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+            fetch-depth: 0 # This is needed to checkout all branches
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Install Yarn
+        run: npm install -g yarn
```

Comment:
Nit: This step should not be necessary; Yarn comes pre-installed on these images

```suggestion
```

Reply:
This is now fixed with this [commit](https://github.com/MetaMask/metamask-extension/pull/19061/commits/3c00775160be618fef4a5e36ba7d6b0d0ba7243d).